devproject/celery.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, unicode_literals
+import os
+from celery import Celery
```

Comment:
There should be a single blank line between `import os` and `from celery` - this is because `os` is standard import and `celery` is 3rd party import.